backend/src/game.ts

```diff
@@ -116,21 +221,28 @@ export default function registerGameRoutes(app: Express) {
 		const index = parseInt(req.params.index);
 		const game = games.get(gameId);
 
+		let answer: number;
+		try {
+			answer = parseInt(req.body.answer);
+		} catch (e) {
+			answer = -1;
+		}
+
```

Comment:
Revert all the changes around checking `answer`. The way it's done on main is correct. See https://github.com/blahajjessie/kakaw/pull/7#discussion_r1176867389.